src/Documentation/Multi-Cluster/MultiClusterConfiguration.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Multi-Clsuter Configuration
+---
+
+## Multi-Cluster Configuration
+
+The multi-cluster configuration determines which clusters are currently part of the multi-cluster. It does not change automatically, but is controlled by the operator. Thus, it is quite different from the membership mechanism used within a cluster, which automatically determines the set of silos that are part of the cluster.
+
+We use the following terminology for the clusters in a service:
+- A cluster is *active* if it has at least one active silo, and *inactive* otherwise
+- A cluster is *joined* if it is part of the current multi-cluster configuration, and *non-joined* otherwise
+
+Being active/inactive is independent from being joined/non-joined: all four combinations are possible. 
+
+All the clusters for a particular service are connected by a [gossip network](GossipChannels.md). The gossip network propagates configuration and status information.  
+
+### Injecting a configuration
+
+An operator issues configuration changes by injecting them into the multi-cluster network. The configurations can be injected into any cluster, and spread from there to all active clusters. Each new configuration consists of a list of cluster ids that form the multi-cluster. It also has a UTC timestamp that is used to track its propagation through the gossip network.
+
+Initially, the multi-cluster configuration is null, which means the multi-cluster list is empty (contains no clusters). Thus, the operator *must* initially inject a multi-cluster configuration. Once injected, this configuration persists in all connected silos (while running) and in all specified gossip channels (if those channels are persistent).
+
+We pose some restrictions on the injection of new configurations that an operator must follow:
+-	Each new configuration may add a number of clusters, or remove a number of clusters (but not both at the same time). 
+-	An operator should not issue a new configuration while a previous configuration change is still being processed.
+
+These restrictions ensure that protocols such as the single-instance-protocol can correctly maintain mutual exclusion of activations even under configuration changes.
+
+#### Via Management Grain
+
+Multi-cluster configurations can be injected on any node in any cluster, using the Orleans Management Grain.
+For example, to inject a multi-cluster configuration that consists of the three clusters { A, B, C }, we can pass a string enumerable to the management grain:
+
+```csharp
+   var clusterlist = "A,B,C".Split(',');
```

Comment:
probably something more relatable is `var clusterList = new[] { "east-us", "west-us", "west-europe" }`